templates/common/_base/files/sysctl-inotify.conf.yaml

```diff
@@ -0,0 +1,8 @@
+filesystem: "root"
+mode: 0644
+path: "/etc/sysctl.d/inotify.conf"
```

Comment:
Nothing here is templated; which means we could ship it by default in RHCOS (in `/usr/lib/sysctl.d`).

This is an interesting topic - one thing to note is that changes made via the MCO only apply when booted as part of a cluster.  So if I just want to e.g. `cosa run` to test the base OS, I'm testing without these types of tweaks.

In this case, maybe that is right - we only want to bump limits once a node has joined the cluster and it's no longer "standalone"?

Anyways for now I'd at least call this `/etc/sysctl.d/rhcos-inotify.conf` to make it clearer where it came from.

Reply:
Although, the other `/etc/sysctl.d/forward.conf` is also generically-named.  So we could also just try to make a cleanup pass on this stuff later and add things like comments to the files describing why they're doing the changes, etc.

Reply:
@cgwalters i tried to be consistent with the other conf, so a full cleanup later with comments, etc.. makes sense to me so things stay consistent.